### PR TITLE
Fix SymbolicRootResource.asFile() returning null and test it

### PR DESCRIPTION
**Pull request by anjumfatima90**
Fix issue #5794 and add tests

### CODE REVIEW

**Comment by anjumfatima90** [comment]
#build

File: dev/com.ibm.ws.kernel.service.location/src/com/ibm/ws/kernel/service/location/internal/SymbolicRootResource.java [hunk 1]
```diff
@@ -270,7 +270,7 @@ public URI toExternalURI() {
     @Override
     @Trivial
     public File asFile() {
-        return null;
+        return root;
```
Comment (tjwatson) [inline dev/com.ibm.ws.kernel.service.location/src/com/ibm/ws/kernel/service/location/internal/SymbolicRootResource.java]: Is there a test for this?

Comment (anjumfatima90) [reply dev/com.ibm.ws.kernel.service.location/src/com/ibm/ws/kernel/service/location/internal/SymbolicRootResource.java]: Added a unit test in com.ibm.ws.kernel.service.location/test/com/ibm/ws/kernel/service/location/internal/SymbolicRootResourceTest.java

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#build

**Comment by anjumfatima90** [comment]
#link 259356

**Pull request merged**